lib/src/runner/configuration/load.dart

```diff
@@ -74,6 +74,30 @@ class _ConfigurationLoader {
   Configuration _loadGlobalTestConfig() {
     var verboseTrace = _getBool("verbose_trace");
     var chainStackTraces = _getBool("chain_stack_traces");
+    var foldStackFrames = _getMap("fold_stack_frames", key: (keyNode) {
+      _validate(keyNode, "fold_stack_frames key must be a string",
```

Comment:
I'd make this 'Must be "only" or "except".', which makes it clearer how to fix the failure. You don't need to include the key name, since the span exception will make it clear where in the file the error is.

This will also let you use the same error for this and for the value check.